WG_JS_CORE.md

```diff
@@ -0,0 +1,110 @@
+# JS Core Dev Team 2019 Roadmap
+
+> Oversees the development of the JS implementation of the IPFS protocol and related projects.
+ 
+**Responsibilities include:**
+- Implement and maintain the IPFS protocol and tools
+- Enable IPFS usage in Node.js, browsers and other JavaScript runtimes
+- Maintain interoperability with other IPFS implementations
+- Incubate and grow projects that will encourage IPFS adoption
+
+**Current Projects**
+- JS IPFS
+- JS IPFS API
+- Go & JS IPFS Interop
+- Npm on IPFS
+- IPFS Service Worker Gateway
+- IPFS Geo IP
+- IPFS Performance Profiling
+- benchmark-js.ipfs.io
+
+
+## 🚀 Major 2019 Goals to achieve
+
+The major goals below and following milestones have been labeled with the emoji for the goal in the IPFS Project Roadmap that they contribute towards:
+ 
+📦 = Package Managers  🗂 = Large Files  🔄 = Decentralized Web  🤝 = Partner Use Cases  🧠 = Strategic Projects
+ 
+- JS IPFS is ready for use in production. It is stable, performant, and interoperable with other IPFS implementations 📦 🗂 🔄 🤝 🧠
+- JS IPFS can ingest and move data with similar speeds to rsync or wget 📦 🗂 🔄 🤝
+- Data on JS IPFS is searchable both online and offline 📦 🔄 🤝
+- Developer experience is a joy  🔄 🤝 🧠
+
+## 💎 Milestones
+
+- `M (P0)`: Production ready 📦 🗂 🔄 🤝 🧠. JS IPFS has for a long time now been under heavy development with a focus on implementing features. We’re now at a stage where it has a good level of feature parity to Go IPFS. Implementing the last key missing features and focused attention on performance, stability and security issues will allow JS IPFS to be a viable alternative to Go IPFS.
```

Comment:
Can these please be sub-headings so that we can link to them?

Reply:
Absolutely. Take full ownership of adjusting the doc as you see fit :)